<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditDeleteEmployee</class>
 <widget class="QWidget" name="FormEditDeleteEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit / Delete Employee</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblEditEmployeeID">
       <property name="text">
        <string>Employee ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditEmployeeID">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblEditEmployeeDate">
       <property name="text">
        <string>Employee Join Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblEditEmployeePosition">
       <property name="text">
        <string>Employee Role</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbEditEmployeePosition"/>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dteEditEmployeeDate"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblEmployeeName">
       <property name="text">
        <string>Employee Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditEmployeeName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblEditEmployeeSalary">
       <property name="text">
        <string>Employee Salary</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinEditEmployeeSalary"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnEditEmployeePrevious">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditEmployeeNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditEmployeeSave">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditEmployeeDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
